doc/source/dev/governance/governance.rst

```diff
@@ -0,0 +1,376 @@
+
+NumFOCUS Subcommittee
+~~~~~~~~~~~~~~~~~~~~~
+
+The Council will maintain one narrowly focused subcommittee to manage
+its interactions with NumFOCUS.
+
+-  The NumFOCUS Subcommittee is comprised of 5 persons who manage
+   project funding that comes through NumFOCUS. It is expected that
+   these funds will be spent in a manner that is consistent with the
+   non-profit mission of NumFOCUS and the direction of the Project as
+   determined by the full Council.
+-  This Subcommittee shall NOT make decisions about the direction, scope
+   or technical direction of the Project.
+-  This Subcommittee will have 5 members, 4 of whom will be current
+   Council Members and 1 of whom will be external to the Steering
+   Council. No more than 2 Subcommitee Members can report to one person
+   through employment or contracting work (including the reportee, i.e.
+   the reportee + 1 is the max). This avoids effective majorities
+   resting on one person.
+
+Institutional Partners and Funding
+==================================
+
+The Steering Council are the primary leadership for the project. No
+outside institution, individual or legal entity has the ability to own,
+control, usurp or influence the project other than by participating in
+the Project as Contributors and Council Members. However, because
+institutions can be an important funding mechanism for the project, it
+is important to formally acknowledge institutional participation in the
+project. These are Institutional Partners.
+
+An Institutional Contributor is any individual Project Contributor who
+contributes to the project as part of their official duties at an
+Institutional Partner. Likewise, an Institutional Council Member is any
+Project Steering Council Member who contributes to the project as part
+of their official duties at an Institutional Partner.
+
+With these definitions, an Institutional Partner is any recognized legal
+entity in the United States or elsewhere that employs at least 1
+Institutional Contributor of Institutional Council Member. Institutional
+Partners can be for-profit or non-profit entities.
+
+Institutions become eligible to become an Institutional Partner by
+employing individuals who actively contribute to The Project as part of
+their official duties. To state this another way, the only way for a
+Partner to influence the project is by actively contributing to the open
+development of the project, in equal terms to any other member of the
+community of Contributors and Council Members. Merely using Project
+Software in institutional context does not allow an entity to become an
+Institutional Partner. Financial gifts do not enable an entity to become
+an Institutional Partner. Once an institution becomes eligible for
+Institutional Partnership, the Steering Council must nominate and
+approve the Partnership.
+
+If an existing Institutional Partner no longer has a contributing
+employee, they will be given a 1 year grace period for remaining
```

Comment:
This is unclear. I think what we are aiming for here is that employees come an go and as long as the gap between employees does not exceed a year, it is OK. One way to say that is that is that the Institutional Partnership will lapse after a period of one year with no active employee.